x-pack/plugins/maps/server/mvt/mvt_routes.ts

```diff
@@ -54,11 +54,10 @@ export function initMVTRoutes({
     ) => {
       const { query, params } = request;
 
-      // todo - replace with direct abortion of raw transport request
-      // const abortController = new AbortController();
-      // request.events.aborted$.subscribe(() => {
-      //   abortController.abort();
-      // });
+      const abortController = new AbortController();
```

Comment:
Do we not need the `AbortController` polyfill for this module? Is that because we aren't referencing the `AbortController.signal`?

Reply:
`AbortController` was introduced in node 15.4.0 about a year ago (https://nodejs.org/api/globals.html#class-abortcontroller). I think the use here in Maps predates that, hence the polyfilll. It's probably no longer required. I'll double check, and make the change. thx!